Doc/library/unittest.rst

```diff
@@ -1278,6 +1310,31 @@ Test cases
 
       .. versionadded:: 3.2
 
+   .. method:: assertStartsWith(s, prefix, msg=None)
+   .. method:: assertNotStartsWith(s, prefix, msg=None)
```

Comment:
```suggestion

   .. method:: assertStartsWith(s, prefix, msg=None)
               assertNotStartsWith(s, prefix, msg=None)
```

And ditto for the rest below (we seem to separate the methods by two blank lines)